---
title: "Sai Zhang - Home"
layout: homelay
excerpt: "Home"
sitemap: false
permalink: /
---

<div class="col-sm-12 clearfix">
<img src="{{ site.url }}{{ site.baseurl }}/images/szhang.jpg" class="img-responsive" width="27%" style="float: left" />
<ul>
<h4><b>Sai Zhang, Ph.D.</b></h4>
Assistant Professor<br>
Department of Epidemiology<br>
Department of Biomedical Engineering (affiliate)<br>
University of Florida<br>
<br>
<b>Office</b>&#58; 2004 Mowry Rd<br>
<b>Phone</b>&#58; (352) 273-5468<br>
<b>Email</b>&#58; sai dot zhang at ufl dot edu<br>
[ <a href="{{ site.url }}{{ site.baseurl }}/CV/cv.pdf">CV</a>, <a href="https://twitter.com/saizhang0">Twitter</a>, <a href="https://www.linkedin.com/in/sai-zhang-8b757817a/">LinkedIn</a> ]
</ul>
</div>

Hi! I am a tenure-track [Assistant Professor](https://epidemiology.phhp.ufl.edu/profile/zhang-sai/) in the [Department of Epidemiology](https://epidemiology.phhp.ufl.edu/) at the [University of Florida (UF)](https://www.ufl.edu/) [College of Public Health & Health Professions (PHHP)](https://phhp.ufl.edu/) and [College of Medicine](https://med.ufl.edu/). I am also an Affiliate Faculty in the [J. Crayton Pruitt Family Department of Biomedical Engineering](https://www.bme.ufl.edu/) at the UF [Herbert Wertheim College of Engineering](https://www.eng.ufl.edu/). I was an Instructor in the [Department of Genetics](https://med.stanford.edu/genetics.html) at [Stanford University School of Medicine](https://med.stanford.edu/). I am also working with Dr. [Philip Tsao](https://profiles.stanford.edu/philip-tsao) as a Research Associate (w/o compensation) at the VA Palo Alto Epidemiology Research and Information Center (ERIC) for Genomics. I received postdoctoral training in Dr. [Michael Snyder](https://med.stanford.edu/snyderlab.html)'s group at Stanford Genetics. Prior to that, I got a Ph.D. in Computer Science from [Tsinghua University](https://www.tsinghua.edu.cn/en/), advised by Dr. [Jianyang Zeng](https://iiis.tsinghua.edu.cn/zengjy/).

Our research lies at the interface of machine learning, genomics, and precision medicine. Our long-term goal is to build machine learning systems to assist scientific discovery, clinical decision making, and personal health management. The focus of our ongoing research is the development of machine learning algorithms (e.g., deep learning and probabilistic graphical models) which exploit massive genetic, multiomic, and clinical data to uncover the genomic basis of complex human diseases. Specifically, our work follows a variant-gene-pathway principle where we start from deep learning modeling of biological sequences (e.g., DNA and RNA) to predict functional effects of variants in different cellular processes (i.e., <i>in silico</i> mutagenesis; [<i>NAR</i> 2016](https://doi.org/10.1093/nar/gkv1025), [<i>Cell Systems</i> 2017](https://doi.org/10.1016/j.cels.2017.08.004), [<i>Bioinformatics</i> 2017](https://doi.org/10.1093/bioinformatics/btx247)). We then move on to a global modeling of genotype-phenotype mapping where we identify candidate risk genes ([<i>Neuron</i> 2022](https://doi.org/10.1016/j.neuron.2021.12.019), [<i>Cell Systems</i> 2022](https://doi.org/10.1016/j.cels.2022.05.007)) and predict phenotypes from personal genomes ([<i>Cell</i> 2018](https://doi.org/10.1016/j.cell.2018.07.021)). By leveraging cutting-edge techniques (e.g., deep learning and single-cell genomics), we are particularly interested in modeling the complexity (e.g., nonlinearity and cell-type-specificity) of the underlying biological system ([<i>Cell</i> 2019](https://doi.org/10.1016/j.cell.2019.03.004)).
<br>
