---
title: "Sai Zhang - Home"
layout: homelay
excerpt: "Home"
sitemap: false
permalink: /
---

<div class="col-sm-12 clearfix">
<img src="{{ site.url }}{{ site.baseurl }}/images/szhang.png" class="img-responsive" width="27%" style="float: left" />
<ul>
<h4><b>Sai Zhang</b></h4>
Instructor<br>
Department of Genetics<br>
Stanford University School of Medicine<br>
<br>
<b>Office</b>&#58; Porter Drive 2270<br>
<b>Phone</b>&#58; (650) 313-9961<br>
<b>Email</b>&#58; zhangsai at stanford dot edu<br>
[ <a href="{{ site.url }}{{ site.baseurl }}/CV/cv.pdf">CV</a>, <a href="https://twitter.com/saizhang0">Twitter</a>, <a href="https://www.linkedin.com/in/sai-zhang-8b757817a/">LinkedIn</a> ]
</ul>
</div>

Hi! I am an [Instructor](https://profiles.stanford.edu/sai-zhang) in the [Department of Genetics](https://med.stanford.edu/genetics.html) at [Stanford University School of Medicine](https://med.stanford.edu/). I received postdoctoral training in Dr. [Michael Snyder](https://med.stanford.edu/snyderlab.html)'s group at Stanford Genetics. Prior to that, I got a Ph.D. in Computer Science from [Tsinghua University](https://www.tsinghua.edu.cn/en/), advised by Dr. [Jianyang Zeng](https://iiis.tsinghua.edu.cn/zengjy/).

Our research lies at the interface of machine learning, genomics, and precision medicine. Our long-term goal is to build machine learning systems to assist scientific discovery, clinical decision making, and personal health management. The focus of our ongoing research is the development of machine learning algorithms (e.g., deep learning and probabilistic graphical models) which exploit massive genetic, multiomic, and clinical data to uncover the molecular mechanisms of human complex diseases. Specifically, our work follows a variant-gene-pathway principle where we start from sequence-based deep learning modeling to predict functional effects of variants in different cellular processes (i.e., <i>in silico</i> mutagenesis; [<i>NAR</i> 2016](https://doi.org/10.1093/nar/gkv1025), [<i>Cell Systems</i> 2017](https://doi.org/10.1016/j.cels.2017.08.004), [<i>Bioinformatics</i> 2017](https://doi.org/10.1093/bioinformatics/btx247)). We then move on to a global modeling of genotype-phenotype mapping where we identify candidate risk genes ([<i>bioRxiv</i> 2020](https://doi.org/10.1101/2020.11.14.382606), [<i>medRxiv</i> 2021](https://doi.org/10.1101/2021.06.15.21258703)) and predict phenotypes from personal genomes ([<i>Cell</i> 2018](https://doi.org/10.1016/j.cell.2018.07.021)). Leveraging cutting-edge techniques (e.g., deep learning and single-cell genomics), we are particularly interested in modeling the complexity (e.g., nonlinearity and cell-type-specificity) of the biological system ([<i>Cell</i> 2019](https://doi.org/10.1016/j.cell.2019.03.004)).
