%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%%%%%%%%%%%%%%%%%%%% Simple LaTeX CV Template %%%%%%%%%%%%%%%%%%%%%%%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%% NOTE: If you find that it says                                     %%
%%                                                                    %%
%%                           1 of ??                                  %%
%%                                                                    %%
%% at the bottom of your first page, this means that the AUX file     %%
%% was not available when you ran LaTeX on this source. Simply RERUN  %%
%% LaTeX to get the ``??'' replaced with the number of the last page  %%
%% of the document. The AUX file will be generated on the first run   %%
%% of LaTeX and used on the second run to fill in all of the          %%
%% references.                                                        %%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%%%%%%%%%%%%%%%%%%%%%%%%%%%% Document Setup %%%%%%%%%%%%%%%%%%%%%%%%%%%%

% Don't like 10pt? Try 11pt or 12pt
%!TEX encoding = UTF-8 Unicode  
\documentclass[10pt]{article}
% The automated optical recognition software used to digitize resume
% information works best with fonts that do not have serifs. This
% command uses a sans serif font throughout. Uncomment both lines (or at
% least the second) to restore a Roman font (i.e., a font with serifs).
\usepackage{times}
\renewcommand{\familydefault}{\sfdefault}

% This is a helpful package that puts math inside length specifications
\usepackage{calc}
\usepackage{comment}

% Simpler bibsection for CV sections
% (thanks to natbib for inspiration)
\makeatletter
\newlength{\bibhang}
\setlength{\bibhang}{1em} %1em}
\newlength{\bibsep}
 {\@listi \global\bibsep\itemsep \global\advance\bibsep by\parsep}
\newenvironment{bibsection}%
        {\begin{enumerate}{}{%
%        {\begin{list}{}{%
       \setlength{\leftmargin}{\bibhang}%
       \setlength{\itemindent}{-\leftmargin}%
       \setlength{\itemsep}{\bibsep}%
       \setlength{\parsep}{\z@}%
        \setlength{\partopsep}{0pt}%
        \setlength{\topsep}{0pt}}}
        {\end{enumerate}\vspace{-.6\baselineskip}}
%        {\end{list}\vspace{-.6\baselineskip}}
\makeatother

% Layout: Puts the section titles on left side of page
\reversemarginpar

%
%         PAPER SIZE, PAGE NUMBER, AND DOCUMENT LAYOUT NOTES:
%
% The next \usepackage line changes the layout for CV style section
% headings as marginal notes. It also sets up the paper size as either
% letter or A4. By default, letter was used. If A4 paper is desired,
% comment out the letterpaper lines and uncomment the a4paper lines.
%
% As you can see, the margin widths and section title widths can be
% easily adjusted.
%
% ALSO: Notice that the includefoot option can be commented OUT in order
% to put the PAGE NUMBER *IN* the bottom margin. This will make the
% effective text area larger.
%
% IF YOU WISH TO REMOVE THE ``of LASTPAGE'' next to each page number,
% see the note about the +LP and -LP lines below. Comment out the +LP
% and uncomment the -LP.
%
% IF YOU WISH TO REMOVE PAGE NUMBERS, be sure that the includefoot line
% is uncommented and ALSO uncomment the \pagestyle{empty} a few lines
% below.
%

%% Use these lines for letter-sized paper
\usepackage[paper=letterpaper,
            %includefoot, % Uncomment to put page number above margin
            marginparwidth=1.3in,     % Length of section titles
            marginparsep=.05in,       % Space between titles and text
            margin=1in,               % 1 inch margins
            includemp]{geometry}

%% Use these lines for A4-sized paper
%\usepackage[paper=a4paper,
%            %includefoot, % Uncomment to put page number above margin
%            marginparwidth=30.5mm,    % Length of section titles
%            marginparsep=1.5mm,       % Space between titles and text
%            margin=25mm,              % 25mm margins
%            includemp]{geometry}

%% More layout: Get rid of indenting throughout entire document
\setlength{\parindent}{0in}

\usepackage[shortlabels]{enumitem}

%% Reference the last page in the page number
%
% NOTE: comment the +LP line and uncomment the -LP line to have page
%       numbers without the ``of ##'' last page reference)
%
% NOTE: uncomment the \pagestyle{empty} line to get rid of all page
%       numbers (make sure includefoot is commented out above)
%
\usepackage{fancyhdr,lastpage}
\pagestyle{fancy}
%\pagestyle{empty}      % Uncomment this to get rid of page numbers
\fancyhf{}\renewcommand{\headrulewidth}{0pt}
\fancyfootoffset{\marginparsep+\marginparwidth}
\newlength{\footpageshift}
\setlength{\footpageshift}
          {0.5\textwidth+0.5\marginparsep+0.5\marginparwidth-2in}
\lfoot{\hspace{\footpageshift}%
       \parbox{4in}{\, \hfill %
                    \arabic{page} of \protect\pageref*{LastPage} % +LP
%                    \arabic{page}                               % -LP
                    \hfill \,}}

% Finally, give us PDF bookmarks
\usepackage{color,hyperref}
\definecolor{darkblue}{rgb}{0.0,0.0,0.3}
\hypersetup{colorlinks,breaklinks,
            linkcolor=darkblue,urlcolor=darkblue,
            anchorcolor=darkblue,citecolor=darkblue}

%%%%%%%%%%%%%%%%%%%%%%%% End Document Setup %%%%%%%%%%%%%%%%%%%%%%%%%%%%


%%%%%%%%%%%%%%%%%%%%%%%%%%% Helper Commands %%%%%%%%%%%%%%%%%%%%%%%%%%%%

% The title (name) with a horizontal rule under it
% (optional argument typesets an object right-justified across from name
%  as well)
%
% Usage: \makeheading{name}
%        OR
%        \makeheading[right_object]{name}
%
% Place at top of document. It should be the first thing.
% If ``right_object'' is provided in the square-braced optional
% argument, it will be right justified on the same line as ``name'' at
% the top of the CV. For example:
%
%       \makeheading[\emph{Curriculum vitae}]{Your Name}
%
% will put an emphasized ``Curriculum vitae'' at the top of the document
% as a title. Likewise, a picture could be included:
%
%   \makeheading[\includegraphics[height=1.5in]{my_picutre}]{Your Name}
%
% the picture will be flush right across from the name.
\newcommand{\makeheading}[2][]%
        {\hspace*{-\marginparsep minus \marginparwidth}%
         \begin{minipage}[t]{\textwidth+\marginparwidth+\marginparsep}%
             {\large \bfseries #2 \hfill #1}\\[-0.15\baselineskip]%
                 \rule{\columnwidth}{1pt}%
         \end{minipage}}

% The section headings
%
% Usage: \section{section name}
\renewcommand{\section}[1]{\pagebreak[3]%
    \hyphenpenalty=10000%
    \vspace{1.3\baselineskip}%
    \phantomsection\addcontentsline{toc}{section}{#1}%
    \noindent\llap{\scshape\smash{\parbox[t]{\marginparwidth}{\raggedright #1}}}%
    \vspace{-\baselineskip}\par}

% An itemize-style list with lots of space between items
\newenvironment{outerlist}[1][\enskip\textbullet]%
        {\begin{itemize}[#1,leftmargin=*]}{\end{itemize}%
         \vspace{-.6\baselineskip}}

% An environment IDENTICAL to outerlist that has better pre-list spacing
% when used as the first thing in a \section
\newenvironment{lonelist}[1][\enskip\textbullet]%
        {\begin{list}{#1}{%
        \setlength{\partopsep}{0pt}%
        \setlength{\topsep}{0pt}}}
        {\end{list}\vspace{-.6\baselineskip}}

% An itemize-style list with little space between items
\newenvironment{innerlist}[1][\enskip\textbullet]%
        {\begin{itemize}[#1,leftmargin=*,parsep=0pt,itemsep=0pt,topsep=0pt,partopsep=0pt]}
        {\end{itemize}}

% An environment IDENTICAL to innerlist that has better pre-list spacing
% when used as the first thing in a \section
\newenvironment{loneinnerlist}[1][\enskip\textbullet]%
        {\begin{itemize}[#1,leftmargin=*,parsep=0pt,itemsep=0pt,topsep=0pt,partopsep=0pt]}
        {\end{itemize}\vspace{-.6\baselineskip}}

% To add some paragraph space between lines.
% This also tells LaTeX to preferably break a page on one of these gaps
% if there is a needed pagebreak nearby.
\newcommand{\blankline}{\quad\pagebreak[3]}
\newcommand{\halfblankline}{\quad\vspace{-0.5\baselineskip}\pagebreak[3]}

% Uses hyperref to link DOI
\newcommand\doilink[1]{\href{http://dx.doi.org/#1}{#1}}
\newcommand\doi[1]{doi:\doilink{#1}}

% For \url{SOME_URL}, links SOME_URL to the url SOME_URL
\providecommand*\url[1]{\href{#1}{#1}}
% Same as above, but pretty-prints SOME_URL in teletype fixed-width font
\renewcommand*\url[1]{\href{#1}{\texttt{#1}}}

% For \email{ADDRESS}, links ADDRESS to the url mailto:ADDRESS
\providecommand*\email[1]{\href{mailto:#1}{#1}}
% Same as above, but pretty-prints ADDRESS in teletype fixed-width font
%\renewcommand*\email[1]{\href{mailto:#1}{\texttt{#1}}}

%\providecommand\BibTeX{{\rm B\kern-.05em{\sc i\kern-.025em b}\kern-.08em
%    T\kern-.1667em\lower.7ex\hbox{E}\kern-.125emX}}
%\providecommand\BibTeX{{\rm B\kern-.05em{\sc i\kern-.025em b}\kern-.08em
%    \TeX}}
\providecommand\Matlab{\textsc{Matlab}}

%%%%%%%%%%%%%%%%%%%%%%%% End Helper Commands %%%%%%%%%%%%%%%%%%%%%%%%%%%

%%%%%%%%%%%%%%%%%%%%%%%%% Begin CV Document %%%%%%%%%%%%%%%%%%%%%%%%%%%%

\begin{document}
\makeheading{SAI ZHANG}

\section{Contact Information}

% NOTE: Mind where the & separators and \\ breaks are in the following
%       table.
%
% ALSO: \rcollength is the width of the right column of the table
%       (adjust it to your liking; default is 1.85in).
%
\newlength{\rcollength}\setlength{\rcollength}{2.5in}%
%
\begin{tabular}[t]{@{}p{\textwidth-\rcollength}p{\rcollength}}
3165 Porter Drive, Room 2270 & +1 (650) 313-9961 \\
Stanford University & \texttt{\email{zhangsai@stanford.edu}} \\
Palo Alto, CA 94304  & \\
\end{tabular}

\section{Research Interests}

Computational Biology, Machine Learning, Genomics, Genetics, Precision Medicine

\section{Research Experience}
\textbf{Stanford University School of Medicine}, Stanford, CA
\begin{outerlist}
\item Instructor, Department of Genetics \hfill Sept. 2021 -- Present
\item Postdoctoral Scholar, Department of Genetics \hfill Sept. 2017 -- Aug. 2021
        \begin{innerlist}
        \item[--] Machine learning-based genome analysis for disease prediction and gene discovery for complex diseases
        \item[--] Advisor: Michael P. Snyder
        \end{innerlist}
\item Visiting Student, Department of Genetics \hfill Dec. 2016 -- Feb. 2017
        \begin{innerlist}
        \item[--] Genome analysis for autism spectrum disorder and abdominal aortic aneurysm
        \item[--] Advisor: Michael P. Snyder
        \end{innerlist}
\end{outerlist}
\vspace{.1in}

\textbf{Palo Alto Veterans Institute for Research}, Palo Alto, CA
\begin{outerlist}
\item Research Associate (without compensation) \hfill Dec. 2019 -- Present
        \begin{innerlist}
        \item[--] Integrative analysis of genetics and electronic medical records for cardiovascular diseases
        \item[--] Advisor: Philip S. Tsao
        \end{innerlist}
\end{outerlist}
\vspace{.1in}

\textbf{Tsinghua University} (THU), Beijing, China
\begin{outerlist}
\item Research Assistant,  Institute for Interdisciplinary Information Sciences (IIIS) \hfill Sep. 2013 -- Jun. 2017
        \begin{innerlist}
        \item[--] Deep learning applications in computational genomics
        \item[--] Advisor: Jianyang Zeng
        \end{innerlist}
\item Research Assistant, Department of Computer Science and Technology \hfill Sep. 2010 -- Jul. 2013
        \begin{innerlist}
        \item[--] Measurement and analysis of online social networks
        \item[--] Advisor: Ke Xu
        \end{innerlist}
\end{outerlist}
\vspace{.1in}

\section{Education}

\textbf{Tsinghua University}, Beijing, China

\begin{outerlist}
\item Ph.D. in
        Computer Science and Technology \hfill Sep. 2013 -- Jun. 2017
        \begin{innerlist}
        \item[--] Thesis: \emph{Deep Learning in Computational Genomics}
        \item[--] Advisor:
            Jianyang Zeng
        \end{innerlist}
\item M.E. in
        Computer Technology \hfill Sep. 2010 -- Jul. 2013
        \begin{innerlist}
        \item[--] Thesis: \emph{Measurement, Analysis and Modeling of Information Propagation in Online Social Networks}
        \item[--] Advisor:
            Ke Xu
        \end{innerlist}
\end{outerlist}

\vspace{.1in}

\textbf{Nanjing University of Science and Technology} (NJUST), Nanjing, China

\begin{outerlist}
\item B.E. in
        Computer Science and Technology \hfill Sep. 2006 -- Jun. 2010
\end{outerlist}

%\section{Research Experience}
%
%\textbf{Research Assistant} \hfill {May 2011 to present}
%\begin{innerlist}
%
%\item[] Division of Biostatistics,\\
%        University of Minnesota\\
%        Supervisor: Bradley P. Carlin, Ph.D
%\end{innerlist}
%\textbf{Research Assistant} \hfill {June 2010 to May 2011}
%\begin{innerlist}
%
%\item[] Division of Epidemiology,\\
%        University of Minnesota\\
%        Supervisors: Traci L. Toomey, Ph.D and Bradley P. Carlin, Ph.D
%\end{innerlist}
%\textbf{Research Assistant} \hfill {Sept 2008 to Aug 2010}
%\begin{innerlist}
%
%\item[] Division of Biostatistics,\\
%        University of Minnesota\\
%        Supervisors: Katherine Huppler-Hullsiek, Ph.D and Jason V. Baker, M.D., M.S.
%\end{innerlist}

\section{Publications}

($^{*}$: equal contribution)

\begin{bibsection}
     \item T.H. Julian, S. Boddy, M. Islam, J. Kurz, K.J. Whittaker, T. Moll, C. Harvey, {\bf S. Zhang}, M.P. Snyder, C. McDermott, J. Cooper-Knock, P.J. Shaw. A review of Mendelian randomization in amyotrophic lateral sclerosis. \emph{Brain}, awab420, 2021.
     \item T. Moll, J.N.G. Marshall, N. Soni, {\bf S. Zhang}, J. Cooper-Knock, P.J. Shaw. Membrane lipid raft homeostasis is directly linked to neurodegeneration. \emph{Essays in Biochemistry}, EBC20210026, 2021. 
     \item J. Cooper-Knock, C. Harvey, {\bf S. Zhang}, T. Moll, I.S. Timpanaro, K.P. Kenna, A. Iacoangeli, J.H. Veldink. Advances in the genetic classification of amyotrophic lateral sclerosis. \emph{Current Opinion in Neurology}, Aug 02, 2021.
     \item T.H. Julian, N. Glascow, A.D.F. Barry, T. Moll, C. Harvey, Y.C. Klimentis, M. Newell, {\bf S. Zhang}, M.P. Snyder, J. Cooper-Knock, and P.J. Shaw. Physical exercise is a risk factor for amyotrophic lateral sclerosis: Convergent evidence from mendelian randomization, transcriptomics and risk genotypes. \emph{EBioMedicine}, 68:103397, 2021.
     \item Y. Li, {\bf S. Zhang}, M.P. Snyder, and K.J. Meador. Precision medicine in women with epilepsy: The challenge, systematic review and future direction. \emph{Epilepsy \& Behavior}, 118:107928, 2021.
     \item X. He, {\bf S. Zhang}, Y. Zhang, Z. Lei, T. Jiang, and J. Zeng. Characterizing RNA pseudouridylation by convolutional neural networks. \emph{Genomics, Proteomics \& Bioinformatics}, 2021.
     \item J. Cooper-Knock, {\bf S. Zhang}, K.P. Kenna, T. Moll, J. Franklin, S. Allen, H.G. Nezhad, N.S. Yacovzada, C. Eitan, E. Hornstein, E. Ehilak, P. Celadova, D. Bose, S.M.K. Farhan, S. Fishilevich, D. Lancet, K.E. Morrison, C. Shaw, A. Al-Chalabi, Project MinE ALS Sequencing Consortium, J.H. Veldink, J. Kirby, M.P. Snyder, and P.J. Shaw. Rare variant burden analysis within enhancers identifies \emph{CAV1} as a new ALS risk gene. \emph{Cell Reports}, 33(9):108456, 2020.
      \item Y. Zhang, {\bf S. Zhang}, X. He, J. Lu, and X. Gao. DeepRibSt: A multi-feature convolutional neural network for predicting ribosome stalling. \emph{Multimedia Tools and Applications}, 1-17, 2020.
     \item J. Li, X. Li, {\bf S. Zhang}, and M.P. Snyder. Gene-environment interaction in the era of precision medicine. \emph{Cell}, 177(1):38-44, 2019.
     \item H. Hu, A. Xiao, {\bf S. Zhang}, Y. Li, X. Shi, T. Jiang, L. Zhanng, L. Zhang, and J. Zeng. DeepHINT: Understanding HIV-1 integration via deep learning with attention. \emph{Bioinformatics}, 35(10):1660-1667, 2019.
     \item J. Li$^{*}$, C. Pan$^{*}$, {\bf S. Zhang}$^{*}$, J.M. Spin, A. Deng, L.L.K. Leung, R.L. Dalman, P.S. Tsao, and M.P. Snyder. Decoding the genomics of abdominal aortic aneurysm. \emph{Cell}, 174(6):1361-1372, 2018.
    \item G. Zhu, W. Deng, H. Hu, R. Ma, {\bf S. Zhang}, J. Yang, J. Peng, T. Kaplan, and J. Zeng. Reconstructing spatial organizations of chromosomes through manifold learning. \emph{Nucleic Acids Research}, 46(8):e50, 2018.
    \item {\bf S. Zhang}, H. Hu, J. Zhou, X. He, T. Jiang, and J. Zeng. Analysis of ribosome stalling and translation elongation dynamics by deep learning. \emph{Cell Systems}, 5(3):212-220, 2017. An early version appeared in \emph{Proceedings of the 21th Annual International Conference on Research in Computational Molecular Biology (RECOMB 2017)}; Hong Kong, China; May 3-7, 2017.
    \item {\bf S. Zhang}, H. Hu, T. Jiang, L. Zhang, and J. Zeng. TITER: Predicting translation initiation sites by deep learning. \emph{Bioinformatics}, 33(14):i234-i242, 2017. Also appeared in \emph{Proceedings of the 25th Conference on Intelligent Systems for Molecular Biology/16th European Conference on Computational Biology (ISMB/ECCB 2017)}; Prague, Czech Republic; Jul. 21-25, 2017.
    \item {\bf S. Zhang}, M. Liang, Z. Zhou, C. Zhang, N. Chen, T. Chen, and J. Zeng. Elastic restricted Boltzmann machines for cancer data analysis. \emph{Quantitative Biology}, 5(2):159-172, 2017. (Cover Article)
     \item S. Li, F. Dong, Y. Wu, {\bf S. Zhang}, C. Zhang, X. Liu, T. Jiang, and J. Zeng. A deep boosting based approach for capturing the sequence binding preferences of RNA-binding proteins from high-throughput CLIP-seq data. \emph{Nucleic Acids Research}, 45(14):e129, 2017.
    \item {\bf S. Zhang}, J. Zhou, H. Hu, H. Gong, L. Chen, C. Cheng, and J. Zeng. A deep learning framework for modeling structural features of RNA-binding protein targets. \emph{Nucleic Acids Research}, 44(4):e32, 2016.
    \item H. Gong, {\bf S. Zhang}, J. Wang, H. Gong, and J. Zeng. Constructing structure ensembles of intrinsically disordered proteins from chemical shift data. \emph{Journal of Computational Biology}, 23(5):300-310, 2016. An early version appeared in \emph{Proceedings of the 19th Annual International Conference on Research in Computational Molecular Biology (RECOMB 2015)}; Warsaw, Poland; Apr. 12-15, 2015.
    \item K. Xu, {\bf S. Zhang}, H. Chen, and H. Li. Measurement and analysis of online social networks. \emph{Chinese Journal of Computers}, 37(1):165-188, 2014. (in Chinese)
\end{bibsection}

% Add a little space to nudge next ``Conference Publications'' marginpar
% down to make room for tall ``Submitted Journal Publications''
% marginpar. If there are enough submitted journal publications, this
% space will not be needed (and should be removed).
%\vspace{0.1in}

\section{Manuscripts \& Preprints}
%\vspace{-.1in}
($^{*}$: equal contribution)
\begin{bibsection}
     \item {\bf S. Zhang}, J. Cooper-Knock, A.K. Weimer, M. Shi, L. Kozhaya, D. Unutmaz, C. Harvey, T.H. Julian, S. Furini, E. Frullanti, F. Fava, A. Renieri, P. Gao, X. Shen, I.S. Timpanaro, K.P. Kenna, J.K. Baillie, M.M. Davis, P.S. Tsao, and M.P. Snyder. Multiomic analysis reveals cell-type-specific molecular mechanisms of COVID-19 severity. \emph{Science Translational Medicine}, under review.
     \item {\bf S. Zhang}, J. Cooper-Knock, A.K. Weimer, C. Harvey, T.H. Julian, C. Wang, J. Li, S. Furini, E. Frullanti, F. Fava, A. Renieri, C. Pan, J. Song, P. Billing-Ross, P. Gao, X. Shen, I.S. Timpanaro, K.P. Kenna, VA Million Veteran Program, GEN-COVID Network, M.M. Davis, P.S. Tsao, and M.P. Snyder. Common and rare variant analyses combined with single-cell multiomics reveal cell-type-specific molecular mechanisms of COVID-19 severity. \emph{medRxiv}, doi: 10.1101/2021.06.15.21258703, 2021.
     \item {\bf S. Zhang}, J. Cooper-Knock, A.K. Weimer, M. Shi, T. Moll, C. Harvey, H.G. Nezhad, J. Franklin, C.D.S. Souza, C. Wang, J. Li, C. Eitan, E. Hornstein, K.P. Kenna, Project MinE Sequencing Consortium, J. Veldink, L. Ferraiuolo, P.J. Shaw, and M.P. Snyder. Genome-wide identification of the genetic basis of amyotrophic lateral sclerosis. \emph{bioRxiv}, doi: 10.1101/2020.11.14.382606, 2020. \emph{Neuron}, under revision.
     \item Y. Wang, X. Zhang, C. Lam, H. Guo, C. Wang, {\bf S. Zhang}, J. Wu, M.P. Snyder, and J. Li. Systems analysis identifies a molecular network in congenital heart diseases. \emph{Circulation}, under review.
     %\item P. Jiang, F. Gao, S. Liu, {\bf S. Zhang}, X. Zhang, Z. Xia, W. Zhang, T. Jiang, J.L. Zhu, Z. Zhang, Q. Shu, M.P. Snyder, and J. Li. Longitudinally tracking personal physiomes for precision management of childhood epilepsy. \emph{Nature Biomedical Engineering}, under review.
     \item {\bf S. Zhang}, K. Xu, X. Chen, and X. Liu. Characterizing information spreading in online social networks. \emph{arXiv}:1404.5562 [cs.SI], 2014.
\end{bibsection}

\section{Teaching Experience}

($^{*}$: graduate course)
\vspace{.1in}

{\bf Teaching Assistant}
\begin{outerlist}
\item Tsinghua University
\begin{innerlist}
\item[--]   Advanced Applications of Machine Learning \hfill Fall 2016
\item[--]   Calculus A II \hfill Spring 2016
\item[--]   Hot Topics in Computational Biology$^{*}$ \hfill Fall 2015
\item[--]   Calculus A II \hfill Spring 2015
\item[--]   Hot Topics in Computational Biology$^{*}$ \hfill Fall 2014
\item[--]   Calculus A II \hfill Spring 2014
\item[--]   Linear Algebra I \hfill Fall 2013
\end{innerlist}
\end{outerlist}

\section{Mentoring Experience}
\begin{outerlist}
\item Omar Nour Niagne (PhD Student, Stanford Genetics) \hfill Jul. 2021 - Present
\item Saahil Jain (MS Student, Stanford CS) \hfill Sept. 2020 - Feb. 2021
\end{outerlist}

\section{Posters}
($^{*}$: equal contribution)
\begin{bibsection}
     \item {\bf S. Zhang}, J. Cooper-Knock, A.K. Weimer, M. Shi, T. Moll, C. Harvey, H.G. Nezhad, J. Franklin, C.D.S. Souza, C. Wang, J. Li, C. Eitan, E. Hornstein, K.P. Kenna, Project MinE Sequencing Consortium, J. Veldink, L. Ferraiuolo, P.J. Shaw, and M.P. Snyder. Genome-wide identification of the genetic basis of amyotrophic lateral sclerosis. \emph{CSHL Meeting: Neurodegenerative Diseases-Biology \& Therapeutics} (Virtual); Dec. 2-4, 2020.
    \item J. Li$^{*}$, C. Pan$^{*}$, {\bf S. Zhang}$^{*}$, J.M. Spin, A. Deng, L.L.K. Leung, R.L. Dalman, P.S. Tsao, and M.P. Snyder. A novel approach for decoding the genomics of a complex disease: Abdominal aortic aneurysm. \emph{16th CEGS Annual Grantee Meeting}; Chicago, IL; Nov. 19-20, 2018.
    \item {\bf S. Zhang}, H. Hu, T. Jiang, L. Zhang, and J. Zeng. TITER: Predicting translation initiation sites by deep learning. \emph{ISMB/ECCB 2017}; Prague, Czech Republic; Jul. 21-25, 2017.
    \item {\bf S. Zhang}, H. Hu, J. Zhou, X. He, T. Jiang, and J. Zeng. ROSE: A deep learning based framework for predicting ribosome stalling. \emph{ISMB/ECCB 2017}; Prague, Czech Republic; Jul. 21-25, 2017.
    \item {\bf S. Zhang}, H. Hu, J. Zhou, X. He, T. Jiang, and J. Zeng. ROSE: A deep learning based framework for predicting ribosome stalling. \emph{IIIS Research Festival 2016}, THU; Dec. 16, 2016.
    \item {\bf S. Zhang}, J. Zhou, H. Hu, H. Gong, L. Chen, C. Cheng, and J. Zeng. Deep learning for modeling structural features of RNA-binding protein targets. \emph{IIIS Research Festival 2015}, THU; Dec. 24, 2015.
\end{bibsection}

\section{Presentations}
\begin{innerlist}
\item Stanford SystemX Alliance 2021 Fall Conference (invited talk, virtual) \hfill Nov. 10, 2021
\item Stanford CEGS GGR Meeting (invited talk, virtual) \hfill Jul. 26, 2021
\item Fred Hutchinson Cancer Research Center (invited talk, virtual) \hfill May 24, 2021
\item Department of Computer Science and Technology, Tsinghua University (invited talk, virtual) \hfill Feb. 23, 2021
\item Illumina AI Lab (invited talk, virtual) \hfill Feb. 9, 2021
\item Regeneron Genetics Center (invited talk, virtual) \hfill Jan. 11, 2021
\item Institute of Modern Biology, Nanjing University (invited talk, virtual) \hfill Nov. 17, 2020
\item 16th CEGS Annual Grantee Meeting (rapid poster talk), Chicago, IL \hfill Nov. 19-20, 2018
\item ISMB/ECCB 2017, Prague, Czech Republic \hfill Jul. 21-25, 2017
\item CSHA/AACR Joint Meeting: Big Data, Computation and Systems Biology in Cancer, Suzhou, China \hfill Dec. 2-5, 2015
\item 2012 CCF National Conference on Service Computing (NCSC 2012), Xi'an, China \hfill Aug. 16-18, 2012
\end{innerlist}

\section{Patents \& Patent Applications}
\vspace{-.1in}
\begin{bibsection}
     \item J. Li, {\bf S. Zhang}, M.P. Snyder, C. Pan, and P.S. Tsao. Processes for genetic and clinical data evaluation and classification of complex human traits. US Patent Application 16/961,120, May 2021.
     \item {\bf S. Zhang}, J. Li, and M.P. Snyder. Systems and methods for predicting genetic diseases. US Patent Application 16/248,314, Jul. 2019.
\end{bibsection}

%\halfblankline

%\section{Presentations}
%Statistical Meetings
%\begin{innerlist}
%\item Workshop on Environmetrics, Raleigh, NC \hfill Oct 2012
%\item Joint Statistical Meetings, San Diego, CA \hfill Aug 2012
%\item Biometric Society (ENAR) Regional Meeting, Washington, D.C. \hfill Apr 2012
%\item Case Studies in Bayesian Statistics and\hfill Oct 2011\\
%Machine Learning, Pittsburgh, PA
%\item Biometric Society (ENAR) Regional Meeting, Miami, FL \hfill Mar 2011
%\item IMS/ISBA Joint International Meeting, Park City, UT \hfill Jan 2011
%\end{innerlist}
%
%\halfblankline
%
%University of Minnesota
%\begin{innerlist}
%\item Mostly Markov Chain Seminar Series \hfill Nov 2011
%\item School of Public Health Research Day \hfill Apr 2011
%\end{innerlist}

\section{Academic Service}
{\bf Invited editor}
\begin{outerlist}
\item Frontiers in Genetics
\end{outerlist}
\halfblankline

{\bf Reviewer}
\begin{outerlist}
    \item Conferences
    \begin{innerlist}
    \item[--] NIPS Workshop on Machine Learning in Computational Biology (MLCB) (2014); International Conference on Intelligent Computing (ICIC) (2018); RECOMB (2018, 2019, 2020, 2021, 2022); ISMB (2020, joint with ECCB in 2019 \& 2021); ACM-BCB (2019)
    \end{innerlist}
    \item Journals
    \begin{innerlist}
    \item[--] Nature Machine Intelligence; Bioinformatics; PLOS Computational Biology; IEEE/ACM Transactions on Computational Biology and Bioinformatics; Briefings in Bioinformatics; Genomics, Proteomics \& Bioinformatics; Scientific Reports; IEEE Journal of Biomedical and Health Informatics; IEEE Transactions on Network Science and Engineering; Journal of Bioinformatics and Computational Biology; Quantitative Biology
    \end{innerlist}
    \item Grants
    \begin{innerlist}
    \item[--] Motor Neurone Disease Association
    \end{innerlist}
%    \item ISMB \hfill 2020
%    \item IEEE Journal of Biomedical and Health Informatics \hfill 2020
%    \item IEEE Journal of Biomedical and Health Informatics \hfill 2020
%    \item IEEE/ACM Transactions on Computational Biology and Bioinformatics \hfill 2020
%    \item Briefings in Bioinformatics \hfill 2020
%    \item IEEE Journal of Biomedical and Health Informatics \hfill 2019
%    \item IEEE Journal of Biomedical and Health Informatics \hfill 2019
%    \item RECOMB 2020 \hfill 2019
%    \item Genes \hfill 2019
%    \item IEEE Journal of Biomedical and Health Informatics \hfill 2019
%    \item IEEE Journal of Biomedical and Health Informatics \hfill 2019
%    \item IEEE Journal of Biomedical and Health Informatics \hfill 2019
%    \item PLOS Computational Biology \hfill 2019
%    \item ACM-BCB \hfill 2019
%     \item Genomics, Proteomics \& Bioinformatics \hfill 2019
%    \item IEEE Transactions on Network Science and Engineering \hfill 2019
%    \item IEEE Journal of Biomedical and Health Informatics \hfill 2019
%    \item Bioinformatics \hfill 2019
%    \item Genomics, Proteomics \& Bioinformatics \hfill 2019
%    \item Genomics, Proteomics \& Bioinformatics \hfill 2019
%    \item ISMB/ECCB \hfill 2019
%    \item Genes \hfill 2019
%    \item Bioinformatics \hfill 2018
%    \item RECOMB 2019 \hfill 2018
%    \item IEEE Journal of Biomedical and Health Informatics \hfill 2018
%    \item IEEE/ACM Transactions on Computational Biology and Bioinformatics \hfill 2018
%    \item The 2018 International Conference on Intelligent Computing (ICIC) \hfill 2018
%    \item PLOS Computational Biology \hfill 2018
%    \item Quantitative Biology \hfill 2018
%    \item RECOMB 2018 \hfill 2017
%    \item Scientific Reports \hfill 2017
    %\item Genomics, Proteomics \& Bioinformatics \hfill 2015
    %\item Journal of Bioinformatics and Computational Biology \hfill 2015
    %\item Workshop on Machine Learning in Computational Biology (MLCB), NIPS \hfill 2014
\end{outerlist}

%\halfblankline

%{\bf Recruiting Committee}
%\begin{outerlist}
%	\item Prepare the entrance exam and interview prospective students for the Zeng Lab \hfill 2014, 2015, 2017
%\end{outerlist}

\section{Awards \& Honors}
\begin{innerlist}
\item Outstanding Graduate, IIIS \hfill 2017
\item Travel Fellowship Award, ISMB/ECCB 2017 \hfill 2017
\item Travel Fellowship Award, RECOMB 2017 \hfill 2017
\item Tsinghua - Baidu Rising Star Scholarship, THU \hfill 2015
\item Outstanding Master's Thesis Award, THU \hfill 2013
\item Outstanding Student Scholarship, NJUST \hfill 2007, 2008, 2009
\item Admitted to the Special Pilot Class, NJUST \hfill 2007
\end{innerlist}

%\section{Memberships}
%\begin{innerlist}
%\item International Society for Computational Biology (ISCB), Student Membership \hfill Mar. 2017 -- Present
%\end{innerlist}

%\section{Skills}
%
%Mathematics
%\begin{innerlist}
%    \item Analysis (real, complex and functional), algebra (linear and abstract), topology (general, differential and algebraic), differential geometry, probability theory, statistics, combinatorics
%\end{innerlist}
%
%Programming Language
%%
%\begin{innerlist}
%    \item Python, MATLAB, C/C$+$$+$, Linux shell script, \LaTeX
%\end{innerlist}

\halfblankline

\section{References}
Johnathan Cooper-Knock
\begin{innerlist}
\item[] NIHR Clinical Lecturer\hfill\\
Department of Neuroscience \hfill {Phone: +44 (0114) 222-2273}\\
The University of Sheffield\hfill E-mail: \texttt{\email{j.cooper-knock@sheffield.ac.uk}}
\end{innerlist}
\halfblankline

%Chao Cheng
%\begin{innerlist}
%\item[] Assistant Professor \\
%Department of Genetics \hfill {Phone: +1 (603) 650-1220}\\
%Geisel School of Medicine at Dartmouth \hfill E-mail: \texttt{\email{chao.cheng@dartmouth.edu}}
%\end{innerlist}

%\halfblankline

%Tao Jiang
%\begin{innerlist}
%\item[] Professor \hfill\\
%Department of Computer Science and Engineering\hfill {Phone: +1 951-827-2991}\\
%University of California, Riverside  \hfill{E-mail: \texttt{\email{jiang@cs.ucr.edu}}}
%\end{innerlist}

\halfblankline
%
%Jian Ma
%\begin{innerlist}
%\item[] Associate Professor \hfill\\
%School of Computer Science\hfill {Phone: +1 (412) 268-2776}\\
%Carnegie Mellon University \hfill{E-mail: \texttt{\email{jianma@cs.cmu.edu}}}
%\end{innerlist}
%
%\halfblankline

Michael P. Snyder
\begin{innerlist}
\item[] W. Ascherman Professor and Chair\hfill\\
Department of Genetics\hfill {Phone: +1 (650) 796-6378}\\
Stanford University \hfill{E-mail: \texttt{\email{mpsnyder@stanford.edu}}}
\end{innerlist}
\halfblankline

Jianyang Zeng
\begin{innerlist}
\item[] Associate Professor \\
Institute for Interdisciplinary Information Sciences \hfill {Phone: +86 (010) 62781693}\\
Tsinghua University\hfill E-mail: \texttt{\email{zengjy321@tsinghua.edu.cn}}
\end{innerlist}

\end{document}

%%%%%%%%%%%%%%%%%%%%%%%%%% End CV Document %%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%----------------------------------------------------------------------%
% The following is copyright and licensing information for
% redistribution of this LaTeX source code; it also includes a liability
% statement. If this source code is not being redistributed to others,
% it may be omitted. It has no effect on the function of the above code.
%----------------------------------------------------------------------%
% Copyright (c) 2007, 2008, 2009, 2010, 2011 by Theodore P. Pavlic
%
% Unless otherwise expressly stated, this work is licensed under the
% Creative Commons Attribution-Noncommercial 3.0 United States License. To
% view a copy of this license, visit
% http://creativecommons.org/licenses/by-nc/3.0/us/ or send a letter to
% Creative Commons, 171 Second Street, Suite 300, San Francisco,
% California, 94105, USA.
%
% THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS
% OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
% MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
% IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY
% CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
% TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
% SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
%----------------------------------------------------------------------%
